packages/trader/src/App/__tests__/app.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from '../app';
+import { mockStore } from '@deriv/stores';
+import moment from 'moment';
+import { TStores } from '@deriv/stores/types';
+
+const root_store = {
+    ...mockStore({}),
+    common: {
+        ...mockStore({}).common,
+        server_time: moment(new Date()).utc(),
+    },
+    client: {
+        ...mockStore({}).client,
+        is_landing_company_loaded: true,
+        is_logged_in: false,
+    },
+    modules: {
+        ...mockStore({}).modules,
+        cashier: {
+            general_store: {
+                onMountCommon: jest.fn(),
+                setAccountSwitchListener: jest.fn(),
+            },
+        },
+    },
+};
```

Comment:
```suggestion
const rootStore = mockStore({
    common: {
        server_time: moment(new Date()).utc(),
    },
    client: {
        is_landing_company_loaded: true,
        is_logged_in: false,
    },
    modules: {
        cashier: {
            general_store: {
                onMountCommon: jest.fn(),
                setAccountSwitchListener: jest.fn(),
            },
        },
    },
});
```
a shorter version :)

Reply:
There are typescript errors if i do it like above because the type is mismatch. I will keep it as it is 🙏

Reply:
@farhan-nurzi-deriv What were the typescript errors? Is it possible to fix those?

Reply:
The error was rootStore type cant be cast to TStores because of some store properties were missing without the `...mockStore({})`. So keeping that in the mock root store object fix the error